KnowledgeBases/TripAdvisor/TripAdvisor.py

```diff
@@ -105,8 +105,9 @@ def getReviews(keyword, place, entityType):
 
 	#check if there are some results specific to the city
 	#Idea in the world queries scope 1.
+	#Trying to support in the world queries
 	scope = int(soup('input', {'id': 'scope'})[0]['value'])
-	if scope == 1:
+	if scope == 1 and 'place' not in ['globe', 'world']:
```

Comment:
I meant place without quotes. Wapas aake karta theek.
On Apr 6, 2016 3:39 PM, "Prabhjyot Singh Sodhi" <notifications@github.com>
wrote:

> In KnowledgeBases/TripAdvisor/TripAdvisor.py
> <https://github.com/Zephrys/Bro/pull/7#discussion_r58681332>:
>
> >  	scope = int(soup('input', {'id': 'scope'})[0]['value'])
> > -	if scope == 1:
> > +	if scope == 1 and 'place' not in ['globe', 'world']:
>
> 'place' not in ['globe', 'world'] will always be true. I'm pretty sure
> you meant some_str['place']
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Zephrys/Bro/pull/7/files/02e00d5ee396d6b50c11903b946bcf077107062e#r58681332>
>